.github/workflows/sync-wiki.yml

```diff
@@ -26,46 +20,13 @@ jobs:
         with:
           repository: "${{ github.repository }}.wiki"
           path: "lichess-bot.wiki"
-      - name: Set path
-        uses: dorny/paths-filter@v3
-        id: changes
-        with:
-          filters: |
-            wiki:
-              - 'wiki/**'
-            home:
-              - 'wiki/Home.md'
-            readme:
-              - 'README.md'
-      - name: Prevent Conflicts
-        if: |
-          steps.changes.outputs.home == 'true' &&
-          steps.changes.outputs.readme == 'true'
-        run: |
-            echo "Error: Conflicting changes. Edit either the README.md file or the wiki/Home.md file, not both."
-            exit 1
       - name: Set github bot global config
-        if: github.event_name == "push"
         run: |
           git config --global user.email "actions@github.com"
           git config --global user.name "actions-user"
-      - name: Sync README.md to wiki/Home.md
-        if: steps.changes.outputs.readme == 'true' && github.event_name == "push"
-        run: |
-          cp -r $GITHUB_WORKSPACE/README.md $GITHUB_WORKSPACE/wiki/Home.md
-          git add wiki/Home.md
-          git commit -m "Auto update wiki/Home.md"
-          git push
-      - name: Sync wiki/Home.md to README.md
-        if: steps.changes.outputs.home == 'true' && github.event_name == "push"
-        run: |
-          cp -r $GITHUB_WORKSPACE/wiki/Home.md $GITHUB_WORKSPACE/README.md
-          git add README.md
-          git commit -m "Auto update README.md"
-          git push
       - name: Sync all files to wiki
-        if: github.event_name == "push"
         run: |
+          cp $GITHUB_WORKSPACE/README.md $GITHUB_WORKSPACE/lichess-bot.wiki
```

Comment:
Shouldn't it be something like `cp $GITHUB_WORKSPACE/README.md $GITHUB_WORKSPACE/lichess-bot.wiki/Home.md`, since GitHub requires a Home page?